.github/workflows/reusable-misc-tests.yml

```diff
@@ -0,0 +1,196 @@
+name: Misc Tests
+
+on:
+  workflow_call:
+    inputs:
+      BUILT_EARTHLY_PATH:
+        required: true
+        type: string
+      BUILD_EARTHLY_TARGET:
+        required: true
+        type: string
+      BINARY:
+        required: true
+        type: string
+      SUDO:
+        type: string
+        required: false
+      RUNS_ON:
+        required: true
+        type: string
+
+
+jobs:
+  misc-tests:
```

Comment:
Is this used anywhere? I think this has been forgotten as I don't see it triggered.

Reply:
Good catch! I've added it now. I definitely missed this one.